Python/sqlmlutils/connectioninfo.py

```diff
@@ -52,10 +52,9 @@ def pwd(self):
 
     @property
     def connection_string(self):
-        return "Driver={driver};Server={server};Database={database};{auth};".format(
-            driver=self._driver,
-            server=self._server if self._port == "" else "{servername},{port}".format(servername=self._server, port=self._port),
-            database=self._database,
-            auth="Trusted_Connection=Yes" if self._uid == "" else
-                 "uid={uid};pwd={pwd}".format(uid=self._uid, pwd=self._pwd)
-        )
+        server=self._server if self._port == "" else f"{self._server},{self._port}"

```

Comment:
This will break support for Python 3.5. Do we have it documented which versions of python this package supports?